tests/new_unit_tests/Commands/Connection/ConnectionCommandTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @file
+ *
+ * Contains Pantheon\Terminus\UnitTests\Commands\Connection\ConnectionCommandTest
+ */
+
+namespace Pantheon\Terminus\UnitTests\Commands\Connection;
+
+
+use ReflectionMethod;
+
+abstract class ConnectionCommandTest extends \PHPUnit_Framework_TestCase
```

Comment:
Yeah I totally agree. I prefer unit tests vs super slow feature tests. I'm just trying to get the process down so we can isolate and execute the command classes. I think a lot of the discussion here is related to that end. Something like a framework functional test that runs via PHPUnit but can do a little more outside-in would be great. Laravel and Symfony have the equivelant of this and I think its a great fit for confirming these methods work. Having something like `TerminusCommandTestCase` to extend from so we can set some stuff up would be pretty cool.